README.md

```diff
@@ -90,7 +88,7 @@ Install this project with:
 ` cd School-library
   - Run gem install after cloning the repo
 
-### Usage
+
 
 
 
```

Comment:
**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍